app/views/summaries/_summaries_table.js.jsx.erb

```diff
@@ -206,6 +185,10 @@
         return s;
       }.bind(this));
 
+      if (render_bonus_deductions == false) {
```

Comment:
In general you shouldn't mutate `props` of a React component. Don't do this check here; instead, do it in the server code, similar to how the `sections` column works.